Assets/MRTK/Services/InputSimulation/SimulatedHandDataProvider.cs

```diff
@@ -177,10 +177,12 @@ public class SimulatedHandDataProvider
 
         internal SimulatedHandState HandStateLeft;
         internal SimulatedHandState HandStateRight;
+        internal SimulatedHandState HandStateGaze;
 
         // If true then hands are controlled by user input
         private bool isSimulatingLeft = false;
         private bool isSimulatingRight = false;
+        private bool isSimulatingGaze => !isSimulatingLeft && !isSimulatingRight;
```

Comment:
I think this might want to be

`private bool isSimulatingGaze => !isSimulatingLeft && !isSimulatingRight && !IsAlwaysVisibleLeft && !IsAlwaysVisibleRight;`

I'm able to get into a slightly weird state when Y or T to toggle the hands `IsAlwaysVisible*`, where the hand is present and its pointers registered, but the simulated GGVPointer is also active. In this state, the gaze cursor disappears since another source is active (and so the gaze pointer is inactive), but I can still click using the simulation.

Reply:
Yeah this didn't break locally and in the initial set of /azp runs. I'll make a PR for this to see if it can fix it.